src/Reader.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Reader {
+
+    private Scanner keyboard;
+    
+    public Reader() {
+        this.keyboard = new Scanner(System.in);
+    }
+
+    public Secret readUserSecret() {
+        String userInput = keyboard.next();
```

Comment:
Use with caution ⚠️ 
next() places the cursor in the same line after reading the input. (it does not include the 'return' character)

https://docs.oracle.com/javase/7/docs/api/java/util/Scanner.html#nextLine()

Reply:
🤔 how would using `next()` bring about unexpected behaviour? I tested the game with both `next()` and `nextLine()` and couldn't see any difference.